routes/quiz.js

```diff
@@ -73,6 +73,9 @@ router.get('/results', function(req, res) {
     else {
       modified_score = quiz_score / 3
     }
+    mod_scores.push(modified_score); // add this quiz's score to the all scores array
```

Comment:
where is mod_scores ever getting more than the single value that is coming right here?